server/mediaserver.go

```diff
@@ -653,19 +657,56 @@ func (s *LivepeerServer) HandlePush(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// Do the transcoding!
-	err = processSegment(cxn, seg)
+	tr, err := processSegment(cxn, seg)
 	if err != nil {
 		// TODO return error
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
 
+	boundary := common.RandName()
+	accept := r.Header.Get("Accept")
```

Comment:
We should probably error out if we see anything other than `multipart/mixed` in this header